_posts/2022-02-16-solidity-0.8.12-release-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.12 Release Announcement'
+date: '2022-02-16'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.12](https://github.com/ethereum/solidity/releases/tag/v0.8.12)
+improves the javascript/wasm binary and fixes several bugs.
+
+## Notable New Features
+
+### Emscripten build / solcjs
+
+The code for the javascript wrappers of the compiler has been partially
+ported to TypeScript, which also impacts some of the aspects of the npm package.
+We hope this is seamless to you but please do not hesitate to provide feedback.
```

Comment:
I'd add a separate header for the typescript migration. These were really two completely independent things.